gcp/README.md

```diff
@@ -77,6 +77,16 @@ Users who already had an RtF email address/Google account usually have performed
 1. `rake plain_sh` is like `rake sh`, but not all configuration is performed. This can be helpful for debugging (e.g. when `rake sh` does not work) and with interactive commands.
 1. To `curl` a single couchdb instance: `kubectl exec --namespace gpii couchdb-couchdb-0 -c couchdb -- curl -s http://$TF_VAR_secret_couchdb_admin_username:$TF_VAR_secret_couchdb_admin_password@127.0.0.1:5984/`
 
+#### Accessing CouchDB and CouchDB Web UI 
```

Comment:
I think this section should go under https://github.com/gpii-ops/gpii-infra/tree/master/gcp#working-with-couchdb-data

Reply:
Done